channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/polling/RequestPoller.kt

```diff
@@ -25,16 +27,17 @@ internal class RequestPoller(
     }
 
     private var unprocessed: Boolean = false
+    private var isActive: Boolean = false
```

Comment:
Shouldn't it be @Volatile in order to ensure the coroutine inside getUpdates always see right value?

Reply:
Volatiles don't work for coroutines correctly. We can use something like `atomicfu`, but I think we don't need it.
These classes are internal and they have single point of interaction (dispatcher). I see no example use case when first thread calls run() and other calls stop() at the same time.

Reply:
What about just AtomicBoolean from java?

Well, yes, it isn't a big problem even if the coroutine will see the updated value a little bit lately, but, anyway, it easy to fix with just AtomicBoolean